llama-index-legacy/llama_index/legacy/embeddings/gemini.py

```diff
@@ -18,6 +18,8 @@ class GeminiEmbedding(BaseEmbedding):
             Defaults to "models/embedding-001".
 
         api_key (Optional[str]): API key to access the model. Defaults to None.
+        api_base (Optional[str]): API base to access the model. Defaults to Official Base.
```

Comment:
pls move the changes to https://github.com/run-llama/llama_index/blob/main/llama-index-integrations/embeddings/llama-index-embeddings-gemini/llama_index/embeddings/gemini/base.py. this file will be deprecated

Reply:
Yea, the proper version does not have this issue actually

Reply:
> pls move the changes to https://github.com/run-llama/llama_index/blob/main/llama-index-integrations/embeddings/llama-index-embeddings-gemini/llama_index/embeddings/gemini/base.py. this file will be deprecated

copy that. updated.